packages/editor/src/functions/browserCheck.ts

```diff
@@ -25,12 +25,7 @@ Infinite Reality Engine. All Rights Reserved.
 
 export const isSupportedBrowser = () => {
   const userAgent = navigator.userAgent as any
```

Comment:
why are we typecasting to `any` type here?
if it comes out to be a separate type, can you try `window.navigator.userAgent`